rslib/src/notetype/schema11.rs

```diff
@@ -162,12 +160,12 @@ impl From<Notetype> for NotetypeSchema11 {
             latex_post: c.latex_post,
             latexsvg: c.latex_svg,
             req: CardRequirementsSchema11(c.reqs.into_iter().map(Into::into).collect()),
-            other,
+            other: bytes_to_other(&c.other),
         }
     }
 }
 
-fn clear_other_duplicates(other: &mut HashMap<String, Value>) {
+fn clear_other_field_duplicates(other: &mut HashMap<String, Value>) {
```

Comment:
Sorry for the mix-up!

Reply:
No problem, it's a bit confusing how there are multiple different `other`s.  :-)